src/api/spec/controllers/webui/obs_factory/staging_projects_controller_spec.rb

```diff
@@ -1,7 +1,14 @@
 require 'rails_helper'
 require 'webmock/rspec'
 
-RSpec.describe Webui::ObsFactory::StagingProjectsController, type: :controller do
+# WARNING: If you change owner tests make sure you uncomment this line
```

Comment:
:thought_balloon: we don't do that anymore. We have a global switch in `spec/support/vcr.rb`

Reply:
Is this documented somewhere? I saw that you guys changed something there, but it's clear for others what changed and how this should be handled now.